blaze/expr/collections.py

```diff
@@ -831,7 +1014,7 @@ def shift(expr, n):
     return Shift(expr, n)
 
 
-dshape_method_list.extend([(iscollection, set([sort, head, tail, sample])),
+dshape_method_list.extend([(iscollection, set([asc, desc, sort, sort2, head, tail, sample])),
```

Comment:
@kwmsmith @llllllllll I think I've addressed all comments except for this one.

 We currently allow ```data.sort_values('a', 'b')```,  which is probably a good shorthand for ```data.sort_values(data.a.asc(), data.b.desc())```.

Do I understand correctly that ```data.sort_values(desc('a'), desc('b'))``` should be disallowed?